config/base/config-leader-election.yaml

```diff
@@ -14,7 +14,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: config-leader-election
+  name: tekton-results-config-leader-election
```

Comment:
Won't this cause issues with the name prefix in the metadata kustomize component? 

See https://github.com/tektoncd/results/blob/main/config/components/metadata/kustomization.yaml#L45 and https://github.com/tektoncd/results/blob/main/config/README.md